content/doc/book/pipeline/multibranch.adoc

```diff
@@ -57,12 +56,12 @@ Upon *Save*, Jenkins automatically scans the designated repository and creates
 appropriate items for each branch in the repository which contains a
 `Jenkinsfile`.
 
-By default, Jenkins will not automatically reindex the repository for branch
+By default, Jenkins will not automatically re-index the repository for branch
```

Comment:
Should be added to the style guide. We don't reindex, we don't reïndex, we re-index!